source/tutorials/php.rst

```diff
@@ -1,193 +1,251 @@
+    This opens the php-fpm.service file for systemd in your editor.
 
-   .. code-block:: bash
+#.  Change the :command:`ExecStart` configuration to add the
+    :command:`--fpm-config` option to point to the custom location:
 
-      sudo systemctl edit --full php-fpm.service
+    .. code-block:: bash
 
-   .. code-block:: console
+       sudo systemctl edit --full php-fpm.service
 
-      [Unit]
-      Description=The PHP FastCGI Process Manager
-      After=syslog.target network.target
+    .. code-block:: console
 
-      [Service]
-      Type=notify
-      PIDFile=/run/php-fpm.pid
-      ExecStart=/usr/sbin/php-fpm --nodaemonize --fpm-config /etc/php.d/php-fpm.conf
-      ExecReload=/bin/kill -USR2 $MAINPID
-      PrivateTmp=true
+       [Unit]
+       Description=The PHP FastCGI Process Manager
+       After=syslog.target network.target
 
-      [Install]
-      WantedBy=multi-user.target
+       [Service]
+       Type=notify
+       PIDFile=/run/php-fpm.pid
+       ExecStart=/usr/sbin/php-fpm --nodaemonize --fpm-config /etc/php.d/php-fpm.conf
+       ExecReload=/bin/kill -USR2 $MAINPID
+       PrivateTmp=true
 
-#. Restart the service
+       [Install]
+       WantedBy=multi-user.target
 
-   .. code-block:: bash
+#.  Restart the service:
 
-      sudo systemctl restart php-fpm.service
+    .. code-block:: bash
 
-#. Verify that the new path has been picked up
+       sudo systemctl restart php-fpm.service
 
-   .. code-block:: bash
+#.  Verify that the new path has been picked up:
 
-      sudo systemctl status php-fpm.service
+    .. code-block:: bash
 
-   You should see the new path in the output
+       sudo systemctl status php-fpm.service
 
-   .. code-block:: console
+    You should see the new path in the output:
 
-      ● php-fpm.service - The PHP FastCGI Process Manager
-      Loaded: loaded (/etc/systemd/system/php-fpm.service; enabled; vendor preset: disabled)
-      Active: active (running) since Thu 2019-10-17 13:19:34 PDT; 8min ago
-      Main PID: 14452 (php-fpm)
-      Status: "Processes active: 0, idle: 0, Requests: 0, slow: 0, Traffic: 0req/sec"
-       Tasks: 1
-      Memory: 11.1M
-      CGroup: /system.slice/php-fpm.service
-              └─14452 php-fpm: master process (/etc/php.d/php-fpm.conf)
+    .. code-block:: console
 
+       ● php-fpm.service - The PHP FastCGI Process Manager
+       Loaded: loaded (/etc/systemd/system/php-fpm.service; enabled; vendor preset: disabled)
+       Active: active (running) since Thu 2019-10-17 13:19:34 PDT; 8min ago
+       Main PID: 14452 (php-fpm)
+       Status: "Processes active: 0, idle: 0, Requests: 0, slow: 0, Traffic: 0req/sec"
+        Tasks: 1
+       Memory: 11.1M
+       CGroup: /system.slice/php-fpm.service
+               └─14452 php-fpm: master process (/etc/php.d/php-fpm.conf)
 
+Background
```

Comment:
@mvincerx  Moved these 2 paras to streamline the config process and created new section to hold this info because it seemed relevant. Look good to you?

Reply:
Very nice! LGTM.